.changeset/young-parents-pump.md

```diff
@@ -0,0 +1,4 @@
+---
+---
```

Comment:
question: Is that intended that it does not touch any packages?

Reply:
Yep, it's only to generate an entry in the changelog. But it shouldn't bump any packages.